Installation/Schema/Repository.schema.json

```diff
@@ -57,6 +57,12 @@
     },
     "default_owner": {
       "type": "string"
+    },
+    "forked_from": {
+      "type": "string"
+    },
+    "archived": {
+      "type": "boolean"
```

Comment:
Same as previous comment

Reply:
Yup, this was already merged with the previous PR, could you merge from master before merging?